docs/sql-guide/functions/function-setcontains.md

```diff
@@ -12,15 +12,15 @@ SETCONTAINS() tests membership of a value within a set expression.
 ## Syntax
 
 ```
-setcontains(set, value)
+setcontains({set, value})
```

Comment:
added brackets because these are both required and so the syntax was wrong

Reply:
We haven't done brackets for any of the sql functions to my knowledge. The only place that exists is for data types.  We say required or not  in the args. We need standards but let's not solve this right now